docs/ubuntu_install.rst

```diff
@@ -1,74 +1,72 @@
+command, we recommend you double check you have entered it correctly before
+pressing enter.
 
-Verify the ``SHA256SUMS`` file and move on to the next step if you see
-"Good Signature" in the output.
+Again, when passing the full public key fingerprint to the ``--recv-key`` command, GPG
+will implicitly verify that the fingerprint of the key received matches the
+argument passed.
 
-::
+.. caution:: If GPG warns you that the fingerprint of the key received
+             does not match the one requested **do not** proceed with
+             the installation. If this happens, please email us at
+             securedrop@freedom.press.
 
-    gpg --verify SHA256SUMS.gpg SHA256SUMS
+Verify the ``SHA256SUMS`` file and move on to the next step if you see
+"Good Signature" in the output. ::
 
-The next and final step is to verify the Ubuntu image. If you are using
-Linux, use the following command.
+    gpg --verify SHA256SUMS.gpg SHA256SUMS
 
-::
+The next and final step is to verify the Ubuntu image. ::
 
     sha256sum -c <(grep ubuntu-14.04.2-server-amd64.iso SHA256SUMS)
 
-If you are using OS X, use the command below.
-
-::
-
-    shasum -a 256 -c <(grep ubuntu-14.04.2-server-amd64.iso SHA256SUMS)
 
 If the final verification step is successful, you should see the
-following output in your terminal.
-
-::
+following output in your terminal. ::
 
     ubuntu-14.04.2-server-amd64.iso: OK
 
+.. caution:: If you do not see the line above it is not safe to proceed with the
+             installation. If this happens, please contact us at
+             securedrop@freedom.press.
+
 Create the Ubuntu installation media
 ------------------------------------
 
-To create the Ubuntu installation media, you can either burn the ISO
-image to a CD-R or create a bootable USB stick (see instructions for
-doing this on `OS
-X <http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-mac-osx>`__,
-`Ubuntu <http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-ubuntu>`__
-and
-`Windows <http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-windows>`__).
-As a reliable method we recommend using the ``dd`` command to copy the
-hybrid ISO directly to a USB drive rather than a utility like UNetbootin
-which can result in errors. Once you have a CD or USB with an ISO image
-of Ubuntu on it, you may begin the Ubuntu installation on both
-SecureDrop servers.
+To create the Ubuntu installation media, you can either burn the ISO image to a
+CD-R or create a bootable USB stick.  As a reliable method we recommend using
+the ``dd`` command to copy the hybrid ISO directly to a USB drive rather than a
+utility like UNetbootin which can result in errors. Once you have a CD or USB
+with an ISO image of Ubuntu on it, you may begin the Ubuntu installation on both
+SecureDrop servers. If your USB is mapped to /dev/sdb, you would use dd like so: ::
+
+    sudo dd conv=fdatasync if=ubuntu-14.04.2-server-amd64.iso of=/dev/sdb
```

Comment:
If we're requiring this happens on Tails and we don't have to document how to do this cross-platform anymore, we should probably detail how to make sure you're writing to the right `of`, since this can be a destructive action.